scripts/export-data.js

```diff
@@ -12,12 +12,24 @@ var regenerate = require('regenerate');
 //              ; SPACEs or TABs at the ends of lines, and is
 //              ; recommended for any character not listed in
 //              ; RFC 2049 as "mail-safe".
+//
+// https://tools.ietf.org/html/rfc2047#section-5 restricts this much
+// more severely in the case quoting is used for a 'word' in an email header:
+//
+//    In this case the set of characters that may be used in a "Q"-encoded
+//    'encoded-word' is restricted to: <upper and lower case ASCII
+//    letters, decimal digits, "!", "*", "+", "-", "/", "=", and "_"
+//    (underscore, ASCII 95.)>.  An 'encoded-word' that appears within a
+//    'phrase' MUST be separated from any adjacent 'word', 'text' or
+//    'special' by 'linear-white-space'.
+//
 var safeSymbols = regenerate()
-	.addRange(33, 60)
-	.addRange(62, 126)
-	// Remove symbols that are unsafe in Q-encoding. Note: space is excluded
-	// because it’s special-cased.
-	.remove('?', '_', '\t');
+	.addRange(65, 90)  // upper case ASCII
+	.addRange(97, 122) // lower case ASCII
+	.addRange(48, 57)  // decimal digits
+	.add('!', '*', '+', '-', '/', '_')
+	// Note: space is included because it’s special-cased.
+	.add(' ');
```

Comment:
Space should continue to be excluded, as it’s already handled here: https://github.com/mathiasbynens/q-encoding/pull/1/files#diff-a148bc52a39c990d02038097d8177455R56